go/store/nbs/store.go

```diff
@@ -896,16 +925,24 @@ func (nbs *NomsBlockStore) GetMany(ctx context.Context, hashes hash.HashSet, fou
 	defer span.End()
 	return nbs.getManyWithFunc(ctx, hashes, gcDependencyMode_TakeDependency,
 		func(ctx context.Context, cr chunkReader, eg *errgroup.Group, reqs []getRecord, keeper keeperF, stats *Stats) (bool, gcBehavior, error) {
-			return cr.getMany(ctx, eg, reqs, found, keeper, nbs.stats)
+			wrappedFound := func(ctx context.Context, c ToChunker) {
+				chk, err := c.ToChunk()
+				if err != nil {
+					// Uh oh. NM4.
+					panic("unexpected error converting chunk to chunk")
+				}
+				found(ctx, &chk)
+			}
```

Comment:
Discussed offline – best to change `getMany` back to delivering chunks, not ToChunker.